README.md

```diff
@@ -25,38 +25,39 @@ Then you have to choose between the different configs, depending on the project
   - react
   - react-native
 
-You'll also need to include eslint & any other plugins/parsers that are needed in your package.json.
-Particularly:
+You'll also need to include eslint & any other plugins/parsers that are needed in your package.json. Particularly:
```

Comment:
But if you want to use the `react` you will also install the `eslint-plugin-react-native` correct?